packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -1,18 +1,68 @@
 import '@testing-library/jest-dom';
 import 'html-validate/jest';
 import { axe, toHaveNoViolations } from 'jest-axe';
-import { render, cleanup } from '../../../../test-utils';
+import { render, screen } from '../../../../test-utils';
 import { ProgressIndicator, ProgressIndicatorProps } from './ProgressIndicator';
 import {
-	ProgressIndicatorItem,
-	ProgressIndicatorItemStatus,
 	activeProgressStatusAllowedList,
 	allProgressStatuses,
-} from './ProgressIndicatorItem';
+	ProgressIndicatorItemStatus,
+	ProgressIndicatorItem,
+} from './utils';
 
 expect.extend(toHaveNoViolations);
 
-afterEach(cleanup);
+function createList(
```

Comment:
Moved this function out for readability. It would be nice to evolve this into a more generic props generator so that it could be used across each test in this file. It'd be even nicer, IMO, to use props generators in all of our tests.